app/controllers/catalog_controller.rb

```diff
@@ -72,7 +72,8 @@ def search_builder processor_chain = search_params_logic
     config.add_facet_field 'date_sim', label: 'Date', limit: 5
     config.add_facet_field 'genre_sim', label: 'Genres', limit: 5
     config.add_facet_field 'collection_ssim', label: 'Collection', limit: 5
-    config.add_facet_field 'unit_ssim', label: 'Unit', limit: 5
+    # The U of A only has one Unit, remove facet
+    # config.add_facet_field 'unit_ssim', label: 'Unit', limit: 5
```

Comment:
In general, I agree that the lines should be deleted. If you checkout the story though, there is some back and forth as to whether this would go ahead, so I would prefer to keep it in commented out as "insurance":

https://github.com/ualbertalib/avalon/issues/17